postgres/scripts/dbdump.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/bash
 
-DBUSER="backup"
-DBNAME="mapswipe"
-DBPASSWD="backupuserpassword"
+SSH_REMOTE_HOST=username@111.11.111.0
+PGUSER="backup"
+PGDATABASE="mapswipe"
+PGPASSWORD="backupuserpassword"
```

Comment:
if these are placeholders, why not, but otherwise, you shouldn't commit passwords